app/templates/bestuurseenheid/zitting/agenda/index.hbs

```diff
@@ -1,29 +1,17 @@
 <div class="container-flex--scroll u-padding--round">
-  <WithRdfaContext @model={{model.agenda}} @class="grid" as |agendaCtx|>
-    <agendaCtx.get @prop="type" @link={{true}} as |elements|>
-      <a {{rdfa elements}} class="u-hidden"></a>
-    </agendaCtx.get>
+  <a href="http://data.lblod.info/id/document-types/agenda" property="dct:type" class="u-hidden"></a>
 
-    <WithRdfaContext @model={{model}} @class="col--7-12 col--9-12--m col--1-1--s" as |ctx|>
-      <h1 class="h1 u-spacer--top--small">
-        <ctx.get @prop="agenda" as |elements|>
-          <span {{rdfa elements}}>Agenda</span>
-        </ctx.get>
-        voor
-        <ctx.get @prop="bestuursorgaan" as |elements _ bInTimeCtx|>
-          <span {{rdfa elements}}>
-            <bInTimeCtx.get @prop="isTijdsspecialisatieVan" as |elements _ bCtx|>
-              <span {{rdfa elements}}>
-                <bCtx.get @prop="naam" />
-              </span>
-            </bInTimeCtx.get>
-          </span>
-        </ctx.get>
-        , zitting van
-        <ctx.get @prop="geplandeStart" as |elements value|>
-          <span {{rdfa elements}} class="u-spacer--tiny">{{moment-format value 'D MMMM YYYY, HH:mm'}}</span>
-        </ctx.get>
-      </h1>
+  <div class="col--7-12 col--9-12--m col--1-1--s" typeof="besluit:Zitting" resource={{model.uri}}>
+    <h1 class="h1 u-spacer--top--small">
+      <span resource={{model.agenda.uri}} typeof="foaf:Document" property="besluit:heeftAgenda">Agenda</span>
+      voor
+      <span resource={{model.bestuursorgaan.uri}} typeof="besluit:Bestuursorgaan" property="besluit:isGehoudenDoor">
+        {{model.bestuursorgaan.isTijdsspecialisatieVan.naam}}
+      </span>
+      , zitting van
+      <span property="besluit:geplandeStart" content={{model.geplandeStart}} datatype="xsd:dateTime" class="u-spacer--tiny">{{moment-format model.geplandeStart 'D MMMM YYYY, HH:mm'}}</span>
```

Comment:
The time in `content={{model.geplandeStart}}` is not formatted correctly. You can use the helper [iso-date](https://github.com/lblod/frontend-gelinkt-notuleren-publicatie/blob/development/app/helpers/iso-date.js)